tests/Http/Controllers/Api/Projects/LargoControllerTest.php

```diff
@@ -443,4 +446,64 @@ public function testStoreJobStillRunningVideoAnnotations()
                 'changed_video_annotations' => [],
             ])->assertStatus(422);
     }
+
+    public function testChunkingImgProject()
+    {
+        config(['biigle.db_param_limit' => 2]);
+        $image = ImageTest::create(['volume_id' => $this->imageVolume->id,
+                                    'filename'=>"testImage"]);
+
+        $imageAnnotations = ImageAnnotation::factory()->count(5)->create(['image_id' => $image->id]); // 70000
+        $label = Label::factory()->create(); 
+
+
+        $imageAnnotations->each(function($imageAnnotation) use($label){
+            ImageAnnotationLabelTest::create([
+                'annotation_id' => $imageAnnotation->id,
+                'user_id' => $this->editor()->id,
+                'label_id' => $label->id,
+            ]);
+        });
```

Comment:
PHP hint: There are also arrow functions which look a little cleaner in cases like these.

```suggestion
        $imageAnnotations->each(fn ($imageAnnotation) => ImageAnnotationLabelTest::create([
            'annotation_id' => $imageAnnotation->id,
            'user_id' => $this->editor()->id,
            'label_id' => $label->id,
        ]));
```